forui/lib/src/widgets/text_field/text_field.dart

```diff
@@ -411,7 +411,10 @@ final class FTextField extends StatelessWidget with FFormFieldProperties<String>
   /// The cursor for a mouse pointer when it enters or is hovering over the widget.
   final MouseCursor? mouseCursor;
 
-  // TODO: InputCounterWidgetBuilder? buildCounter;
+  /// The [InputCounterWidgetBuilder] to build a custom counter for the text field.
```

Comment:
nitpick: I think we should introduce a `FTextFieldCounterBuilder` typedef that's a copy of the `InputCounterWidgetBuilder`.  Reason being that `InputCounterWidgetBuilder` is part of material, and we generally don't expose material types in Forui.